librad/src/git/storage.rs

```diff
@@ -145,7 +150,9 @@ pub enum RadSelfSpec {
 }
 
 pub type NoSigner = PhantomData<!>;
-pub type WithSigner = SecretKey;
+
+pub trait WithSigner: sign::Signer + Send + Sync + Clone + 'static {}
```

Comment:
The name seems a bit strange. Why not just `Signer`?

Reply:
Ya, I was sticking with naming. I think before it made sense because the mnemonic `Storage<WithSigner>` made sense.

Reply:
But it makes sense to change it :)